Dockerfile.test

```diff
@@ -0,0 +1,27 @@
+# Take in a runtime image to use for the base system
+# Expects an alpine image
+ARG RUNTIME_IMAGE
+
+# Use the docker image provided via build arg
+FROM $RUNTIME_IMAGE
+
+# Install the libraries we need for go and dockerize
+RUN apk add --no-cache curl git
+
+# Copy in the dockerize utility
+ARG DOCKERIZE_VERSION=0.6.0
+RUN curl -sL https://github.com/jwilder/dockerize/releases/download/v$DOCKERIZE_VERSION/dockerize-alpine-linux-amd64-v$DOCKERIZE_VERSION.tar.gz | tar -xzC /usr/local/bin
+
+# Copy project into the image
+COPY . /src/github.com/fauna/faunadb-go
+
+# Shift over to the project, set GOPATH, and fetch dependencies
+WORKDIR /src/github.com/fauna/faunadb-go/faunadb
+ENV GOPATH=/
```

Comment:
is this common practice to set a gopath of `/` in a docker image? My gut would say that we'd have it in like `/var/lib/go` or `/opt/go` or `/usr/local/go` or somewhere

Reply:
Probably not, but I kinda threw up my hands at this point and just went for the first thing that worked :| I will update this.